### DISCUSSION TITLE
add optin to change this message:  /Trust <player>  Grants a player permission to build.  See also /UnTrust, /ContainerTrust, /AccessTrust, and /PermissionTrust.

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1662] [Ideas]
![image](https://user-images.githubusercontent.com/75040244/142090932-02aa3056-f287-4969-8f3c-5cae6aa546e2.png)

**Comment by bigpresh** [comment]
Looks like that's from Bukkit help output, which comes from [plugin.yml](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/resources/plugin.yml) so not sure how easy changing that at runtime would be.

**Reply by thiagogebrimm** [reply]
If you could change it through the GriefPrevention folder, it would be easier
